pages/common/kate.md

```diff
@@ -0,0 +1,32 @@
+# kate
+
+> KDE Text Editor.
+> More information: <https://kate-editor.org/>.
+
+- Launch Kate and open specific files:
+
+`kate {{path/to/file1}} {{path/to/file2}}`
+
+- Open a remote document in Kate:
+
+`kate {{https://example.com}}`
+
+- Start a new Kate instance:
+
+`kate --new`
+
+- Open a file in Kate with the cursor at the specific line:
+
+`kate --line {{line_number}} {{path/to/file}}`
+
+- Delete the files opened by Kate after use:
```

Comment:
```suggestion
- Delete the files created by Kate after use:
```
Is that how it works? I don't assume Kate deletes an already exesting files after you edited it

Reply:
```
When used, the specified files are treated as temporary files and deleted (if they are local files and you have sufficient permissions) when closed, unless they were modified since they were opened.
```

Reply:
I don't get how this could be useful? Why would you want to open a file, look at it and then delete it?

Maybe, I'm also missing something, because Kate didn't delete anything when I tried it.

Reply:
I'll replace this example :joy: